db/seeds.rb

```diff
@@ -1,15 +1,39 @@
-require 'factory_bot'
+ADRESSES = [
```

Comment:
Why constant?

Reply:
because,Faker isn't valid address,I discussed this with Anton